updates/2024-02-12-editorial-board-update.md

```diff
@@ -0,0 +1,119 @@
+# Meeting Notes: Python Docs Editorial Board
+
+## February 12, 2024
+
+## Attendees
+
+Mariatta, Ned, Joanna, Guido
+
+
+## Agenda
+
+1. What project do we want to deliver first? Here are some ideas to start the discussion:
+
+a. Define scope and outline for new intro tutorial
+b. Define scope and outline for new landing page
+c. Inventory/audit a section of the docs
+d. Personas (see #3 below)
+
+
+2. What does the docs community need from us most right now? Here are some ideas to start the discussion:
+
+- How-to discussion
+- First-person discussion 
+- A more thorough and more skimmable style guide
+
+3. [meta] Do we have a way to collect the largish topics they’ve discussed that we could provide guidance for? -> see editorial-board GH repo
+
+- Function parameters: bulleted or not
+- Examples: how to integrate into the docs
+- Type hints 
+
+4. Do we have an idea of what type of learner/user we need to optimize the docs for?
+
+- Wait until Carol is back 
+
+5. Do we have an idea of what type of contributor we want to prioritize removing obstacles for and creating systems for?
+
+6. Will the editorial board give an update at the language summit or PyCon?
+
+7. The language summit isn’t even announced yet. We’ll think about it next time
```

Comment:
```suggestion
6. Will the editorial board give an update at the language summit or PyCon?

   - The language summit isn’t even announced yet. We’ll think about it next time
```
This sounds like an answer to the previous item rather than its own item.